teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, parent_permission, name, specialization)
```

Comment:
- [ ]  The name' and 'parent_permission'  should have default values of "Unknown" and 'true', please add it